src/components/skip-to-content/_skip.scss

```
@@ -14,7 +14,7 @@
 
   &:focus {
     background-color: var(--ons-color-focus);
-    box-shadow: 0;
+    box-shadow: 0;        
```

```suggestion
    box-shadow: 0;
```
Should get rid of this extra spacing thats been added. Other than that looks good!